views/pages/listaAcolhidos.ejs

```diff
@@ -4,10 +4,10 @@
         <h1 class="titulo-lista-acolhidos">Lista de Acolhidos</h1>
         <div class="content">
             <ul class="lista-acolhidos">
-                <%for (users of users){%>
+                <%for (acolhidos of acolhidos){%>
```

Comment:
Neste caso não seria `acolhido of acolhidos`? Já que o lado esquerdo do `of` representa somente um item do array a cada iteração.